dev/data/fineweb.py

```diff
@@ -61,16 +65,33 @@
     name = "edu_fineweb"
 
 # init the tokenizer
-enc = tiktoken.get_encoding("gpt2")
-eot = enc._special_tokens['<|endoftext|>'] # end of text token
-def tokenize(doc):
+def tokenize(doc, model):
```

Comment:
instead of taking model and having a big `if` inside `def` we can have two defs for the two options, and dispatch accordingly

Reply:
Sure, at this point, I don't have a special preference as we only support 2 models.